package.json

```diff
@@ -307,6 +308,7 @@
     "less-loader": "^4.0.5",
     "mkdirp": "^0.5.1",
     "mz": "^2.7.0",
+    "npm-force-resolutions": "0.0.3",
```

Comment:
This shouldn't be necessary. The `npx` command runs a binary in an npm repo (in this case the `npm-force-resolutions`). It should download the repo and run the necessary command on-demand. Brave itself doesn't need to have the dependency in its source tree as far as I can tell.

Reply:
right - but preinstall is running `npx npm-force-resolutions`. If we don't add this, nothing would happen? 🤔